src/functionalTest/groovy/com/autonomousapps/AbstractFunctionalSpec.groovy

```diff
@@ -24,9 +22,7 @@ abstract class AbstractFunctionalSpec extends Specification {
   // For faster CI times, we only test min + max. Testing all would be preferable, but we don't have till the heat death
   // of the universe to wait.
   protected static final SUPPORTED_GRADLE_VERSIONS = [
-    GRADLE_7_0,
-//    GRADLE_7_1,
-//    GRADLE_7_2,
+    GRADLE_7_2,
```

Comment:
Note to self: update wiki to note that the new minimum required version of Gradle is now 7.2.